src/server/utils/util.ts

```diff
@@ -43,3 +43,9 @@ export const månedÅrTilDate = (årMåned: string): Date => {
 
 export const formaterBeløp = (verdi: number): string =>
   Number(verdi).toLocaleString('no-NO', { currency: 'NOK' }) + ' kr';
+
+const mapTrueFalse = (bool?: boolean): string =>
+  bool === true ? 'Ja' : bool === false ? 'Nei' : '';
```

Comment:
```suggestion
const mapTrueFalseTilString = (bool: boolean): string =>  bool ? 'Ja' : 'Nei';
```
Tror dette gir samme oppførsel hvis du tar med endringen under også